src/soc/intel/baytrail/bootblock/microcode_asm.S

```diff
@@ -0,0 +1,128 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
I decided not to modify the common file, because:

1. I would have to think how to compile the file with CONFIG_CBFS_VERIFICATION and avoid a situation when walkcbfs is not compiled and the code location is also not provided...
2. The code could use CPU_MICROCODE_CBFS_LOC but it is for CBFS, so won't work.
3. I would have to add my EARLY_CPU_MICROCODE_LOC

I didn't want to make it even more messy...